source/renderer/app/domains/News.js

```diff
@@ -170,6 +176,10 @@ class NewsCollection {
     // Order read from newest to oldest
     return orderBy(read, 'date', 'asc');
   }
+
+  // @computed get update(): News | null {
```

Comment:
@daniloprates should this be removed?

Reply:
[Yes](https://github.com/input-output-hk/daedalus/blob/da2a056e37b83cc33e94aa25a3d3faadea76917c/source/renderer/app/domains/News.js#L78)